### DISCUSSION TITLE
Where can vscode view the default colors for parts that you don't define

### DISCUSSION THREAD

**Discussion by QingyaoLin** [microsoft/vscode-discussions#391] [Extension Development QnA] [answered]
I sometimes wonder what vscode's default widget color is. For example: when I use a github theme, I find that it only defines one part of the color internally, so where is the other part defined, and how do I view it.

For example, when I look at the color scheme of the topic by `Developer: generator color theme...` command, the undefined part is always shown in the form of comments(e.g `"activityBar.activeBackground": null,`). How can I see the color of this annotated section?

**Comment by isidorn** [comment]
@aeschli might know

**Comment by aeschli** [comment, marked as answer]
If a theme does not define a color, the default color is used. And default colors can be `null`
It depends on each component's code on what happens when a color is null. 
You would have to use the Inspect tool of Developer Tools to find out more, but components are free to change that behavior (or the default).
Ideally the theme defines all colors.

**Reply by QingyaoLin** [reply]
Thank you for your answer.
